docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -279,11 +305,14 @@ vars from a JSON or YAML file
 Variable precedence: Where should I put a variable?
 ===================================================
 
-You can set multiple variables with the same name in many different places. When you do this, Ansible loads every possible variable it finds, then chooses the variable to apply based on variable precedence. In other words, the different variables will override each other in a certain order.
+You can define multiple variables with the same name in many different places. When you do this, Ansible loads every possible variable it finds, and then chooses the variable to apply based on the variable precedence. In other words, the different variables will override each other based on how they are arranged. Therefore, you can control Ansible behavior simply by arranging the order of variables.
+
+In addition to the arrangement of variables, the other things that can affect Ansible behavior are Ansible configuration, command-line options, and playbook keywords.  In general, variables take precedence, so that host-specific settings can override the more general settings. For examples and more details on the precedence of these various settings, see :ref:`general_precedence_rules`.
```

Comment:
Hm, this is another place where the suggested change is drawing attention to ambiguous or unclear original text. Maybe something like this:
```suggestion
Some behavioral parameters can be set in variables and also in Ansible configuration, as command-line options, and using playbook keywords.  In general, if the same parameter is set by a variable and by another method, the variable takes precedence, so that host-specific settings can override the more general settings. For examples and more details on the precedence of these various settings, see :ref:`general_precedence_rules`.
```